public/pages/workflow_detail/workflow_inputs/processor_inputs/input_transform_modal.tsx

```diff
@@ -59,10 +66,26 @@ export function InputTransformModal(props: InputTransformModalProps) {
 
   // source input / transformed output state
   const [sourceInput, setSourceInput] = useState<string>('[]');
-  const [transformedOutput, setTransformedOutput] = useState<string>('[]');
+  const [transformedOutput, setTransformedOutput] = useState<string>('{}');
 
   // get the current input map
-  const map = getIn(values, `ingest.enrich.${props.config.id}.inputMap`);
+  const map = getIn(
+    values,
+    `ingest.enrich.${props.config.id}.inputMap`
+  ) as MapArrayFormValue;
+
+  // selected output state
+  const outputOptions = map.map((_, idx) => ({
+    value: idx,
+    text: `Prediction ${idx + 1}`,
+  })) as EuiSelectOption[];
+  const [selectedOutputOption, setSelectedOutputOption] = useState<
+    number | undefined
+  >(
+    get(outputOptions, '0.value') !== undefined
+      ? (outputOptions[0].value as number)
+      : undefined
```

Comment:
nit - same as a couple of occurrences below: this can be simplified as `outputOptions[0]?.value ?? undefined`

Reply:
sure, will update.